crates/autopilot/src/run_loop.rs

```diff
@@ -166,9 +165,7 @@ impl RunLoop {
 
             let mut prices = BTreeMap::new();
             let mut fee_policies = Vec::new();
-            let block_deadline = competition_simulation_block
-                + self.submission_deadline
-                + self.additional_deadline_for_rewards;
+            let block_deadline = competition_simulation_block + self.submission_deadline;
```

Comment:
`additional_deadline_for_rewards` has `5` default value. The block deadline is reduced by that value. Is it fine?

Reply:
@squadgazzz the block deadline is not used anywhere as I mentioned in the description. It is used to just store it in the competition. What's more, the overall idea with this change (and the upcoming one) is to reduce it to 3 blocks.